backend/tests/regression/answer_quality/api_utils.py

```diff
@@ -58,8 +57,10 @@ def get_answer_from_query(query: str, run_suffix: str) -> tuple[list[str], str]:
         context_data_list = response_json.get("contexts", {}).get("contexts", [])
         answer = response_json.get("answer", "")
     except Exception as e:
-        print("Failed to answer the questions, trying again")
-        print(f"error: {str(e)}")
+        print("Failed to answer the questions:")
+        print(f"\t {str(e)}")
+        print("Restarting vespa container and trying agian")
+        restart_vespa_container(run_suffix)
```

Comment:
we now restart the vespa container to fix connectivity issues as that was the only thing 

it would probably be worthwhile to investigate why this is happening and how to fix it long term to improve overall product stability